Dockerfile

```
@@ -163,12 +151,17 @@ RUN \
         pulseaudio-alsa \
         socat
 
+WORKDIR /tmp/
+COPY requirements.txt /tmp/requirements.txt
```

I think we don't need to change workdir when pip-installing requirements from an absolute path. Tested and builds fine. Also saves one layer.
```suggestion
COPY requirements.txt /tmp/requirements.txt
```